module/applications/actor/character-sheet-2.mjs

```diff
@@ -198,6 +198,19 @@ export default class ActorSheet5eCharacter2 extends ActorSheet5eCharacter {
       path: showTokenPortrait ? this.actor.isToken ? "" : "prototypeToken.texture.src" : "img"
     };
 
+    // Encumbrance
+    const encumberedThreshold = game.settings.get("dnd5e", "metricWeightUnits") 
+      ? CONFIG.DND5E.encumbrance.threshold.encumbered.metric 
+      : CONFIG.DND5E.encumbrance.threshold.encumbered.imperial;
+    const heavilyEncumberedThreshold = game.settings.get("dnd5e", "metricWeightUnits") 
+      ? CONFIG.DND5E.encumbrance.threshold.heavilyEncumbered.metric
+      : CONFIG.DND5E.encumbrance.threshold.heavilyEncumbered.imperial;
+    const maximumThreshold = game.settings.get("dnd5e", "metricWeightUnits")
+      ? CONFIG.DND5E.encumbrance.threshold.maximum.metric
+      : CONFIG.DND5E.encumbrance.threshold.maximum.imperial;
+    context.encumbrance.low = (encumberedThreshold / maximumThreshold) * 100;
+    context.encumbrance.high = (heavilyEncumberedThreshold / maximumThreshold) * 100;
+
```

Comment:
We are already calculating the stops in `system.attributes.encumbrance.stops`, so you can skip this whole section.

```suggestion
```

Reply:
It looks like the stops are only updated with an actor update, rather than on opening the character sheet or following an up to the encumbrance thresholds config. Definitely worthwhile using the existing calculation, but think it needs extra development to make the sheet instantly reflective of the config to avoid confusion for users.

Reply:
I'm not sure how important it is to do this dynamically, and if a module adds it to a UI they can just manually call `reset` on any actor when the value is changed.